src/e-auto-update.js

```diff
@@ -122,8 +122,8 @@ function checkForUpdates() {
     if (headBefore === git(headCmd)) {
       console.log('build-tools is up-to-date');
     } else {
-      console.log(color.childExec('npx', ['yarn', '--prod'], execOpts));
-      cp.execSync('npx yarn --prod', execOpts);
+      console.log(color.childExec('npx', ['yarn', '--immutable'], execOpts));
+      cp.execSync('npx yarn --immutable', execOpts);
```

Comment:
`npx` is unsafe and now we have the yarn cjs file we can just do `node .yarn/releases/yarn-4.10.3.cjs` instead.

I'm also fairly sure this will fail on the first run through because `yarn --prod` will run from the _old_ script while having the new code locally

Reply:
I think we should be OK to just Let This Happen.

We can post in #pineapple internally that this will happen, it will auto-fix on the next update. People won't be broken, they'll get an error _once_ and then it will fix on the next `e` command they run